includes/payment-methods/class-upe-payment-gateway.php

```diff
@@ -948,6 +960,7 @@ public function get_upe_available_payment_methods() {
 		$methods[] = 'sofort';
 		$methods[] = 'sepa_debit';
 		$methods[] = 'p24';
+		$methods[] = Link_Payment_Method::PAYMENT_METHOD_STRIPE_ID;
```

Comment:
A great reuse of the constant 🙌

Reply:
I wish all other payment methods had constant names. Thanks for taking care of CC.